backend/core/database.py

```diff
@@ -831,7 +831,9 @@ def handle_hijack_ongoing_request(self, message):
                 try:
                     results = []
                     query = (
-                        "SELECT DISTINCT ON(h.key) b.key, b.prefix, b.as_path, b.type, h.key, h.hijack_as, h.type "
+                        "SELECT b.key, b.prefix, b.origin_as, b.as_path, b.type, b.peer_asn, "
```

Comment:
Now we return ALL BGP updates of each hijack (instead of one per hijack), since in the case it is a hijack after the outdate, all related updates need to be re-examined and go to a new hijack alert. This is critical, please examine carefully. Instead of getting one update per hijack we now get all updates of each hijack to examine (which is the correct way, even if it hurts performance in a few extraordinary cases).